bazel/src/main/java/com/google/api/codegen/bazel/BUILD.bazel.raw_api.mustache

```diff
@@ -32,10 +32,13 @@ java_proto_library(
     deps = [":{{name}}_proto"],
 )
 
-java_grpc_library(
-    name = "{{name}}_java_grpc",
-    srcs = [":{{name}}_proto"],
-    deps = [":{{name}}_java_proto"],
+# Open Source Packages
+java_gapic_assembly_gradle_pkg(
+    name = "{{type_only_assembly_name}}-java",
+    deps = [
+        ":{{name}}_proto",
```

Comment:
It's probably my own lack of understanding, why do we need this for Java but not for other languages?

Reply:
No idea tbh. I just saw it in the same target for GAPIC libraries and copied.

Reply:
Perhaps because the protos are included in the generated code under the `proto-` directory?